src/common/ActionButtons/ActionButton.tsx

```diff
@@ -89,7 +90,7 @@ const ActionButton: FC<ActionButtonProps> = props => {
 			aria-label={ariaLabel}
 		>
 			<span dangerouslySetInnerHTML={{ __html }} />
-			{isWebURL && <LinkIcon />}
+			{icon ? icon : isWebURL ? <LinkIcon /> : null}
```

Comment:
nested ternary operator is unreadable, please move to a more explicit variable above return statement

Reply:
fixed and also I added a new property to exclude the icon also if is a web_url type. It can be useful in the future